### PR TITLE
SC-6169: Adjust entry-point with an additional Ip's 

### PR DESCRIPTION
**Pull request by alexanderM91**
### Description

- Ticket/Issue: https://spryker.atlassian.net/browse/SC-6169

#### Change log

- 
### Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: generator/src/templates/nginx/entrypoint.sh.twig [hunk 1]
```diff
@@ -9,6 +9,17 @@ export SPRYKER_NGINX_CGI_{{ applicationName | upper}}=${SPRYKER_NGINX_CGI_{{ (ap
 {% endfor %}
 {% endfor %}
 
+export ALLOWED_IP="${ALLOWED_IP}:-127.0.0.1"
```
Comment (sprymiker) [inline generator/src/templates/nginx/entrypoint.sh.twig:12]: Wrong syntax

**Comment by sprymiker** [comment]
Consider checking this IPs (at least one) in local env by setting it in docker-compose.yml pointing to internal Docker gateway IP.

File: generator/src/templates/nginx/entrypoint.sh.twig (hunk 1 above)
Comment (alexanderM91) [reply generator/src/templates/nginx/entrypoint.sh.twig:12]: Updated

**Comment by sprymiker** [comment]
@Bakdark Do we need this code or everything is solved on the Cloud side?

**Comment by alexanderM91** [comment]
Discussed with @Bakdark.
The current PR can be closed.

**Pull request closed without merging**